RCTMapboxGL/RCTMapboxGL.m

```diff
@@ -247,6 +228,17 @@ - (BOOL)mapView:(RCTMapboxGL *)mapView annotationCanShowCallout:(id <MGLAnnotati
     return YES;
 }
 
+- (void)selectAnnotationAnimated:(NSUInteger)annotationInArray
+{
+    if (_annotations) {
```

Comment:
If `-[RCTMapboxGL setAnnotations:]` is never called, `-selectAnnotationAnimated:` does nothing. Should it assert?